### PR TITLE
SAIC-205: Make users migration optional

### PR DESCRIPTION
**Pull request by miarmak**
Patch provides possibility to enable/disable users migration in the config.

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSED.

File: devlab/config.template [hunk 1]
```diff
@@ -16,6 +16,7 @@ file_compression = gzip
 level_compression = 9
 overwrite_user_passwords = False
 migrate_quotas = True
+migrate_users = False
```
Comment (roman-verchikov) [inline devlab/config.template]: I guess it should be `true` for devlab

**Comment by roman-verchikov** [comment]
Looks good other than 1 inline comment

File: devlab/config.template (hunk 1 above)
Comment (miarmak) [inline devlab/config.template]: Okay

**Comment by Wasapl** [comment]
Merged build finished. Test PASSED.

**Comment by plomakin** [comment]
+1

**Comment by roman-verchikov** [comment]
+1

**Comment by roman-verchikov** [comment]
Got two +1s, can be merged

**Pull request merged**